docs/lib/content/configuring-npm/package-json.md

```diff
@@ -617,6 +617,7 @@ See [semver](https://github.com/npm/node-semver#versions) for more details about
 * `tag` A specific version tagged and published as `tag`  See [`npm
   dist-tag`](/commands/npm-dist-tag)
 * `path/path/path` See [Local Paths](#local-paths) below
+* `npm:@scope/pkg@version` Custom alias for a pacakge See [`npm install`](/commands/npm-install) 
```

Comment:
Do you think `/using-npm/package-spec#aliases` would be a better place to link? https://docs.npmjs.com/cli/v10/using-npm/package-spec#aliases

Reply:
certainly yes